### ISSUE TITLE
mysql-client is not available no longer

### ISSUE THREAD

**Issue by nagaakihoshi** [circleci/circleci-docs#3536]
# Locations
https://circleci.com/docs/2.0/postgres-config/#example-mysql-project

```
sudo apt-get install mysql-client
```

# Details

Current result

```
Package mysql-client is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'mysql-client' has no installation candidate
```

I should set another package 'default-mysql-client' according to official guide.

https://packages.debian.org/stable/database/

So rewrite the line as...

```
sudo apt-get install default-mysql-client
```

**Comment by nagaakihoshi** [comment]
Create a pull request as well.

https://github.com/circleci/circleci-docs/pull/3537

**Comment by teesloane** [comment]
Ah! Thank you for catching this. It looks like it's related to our [image updates](https://discuss.circleci.com/t/docker-convenience-images-updates-july-2019/31239/5). Thanks for putting up a PR as well. I will go ahead and approve that.

**Comment by teesloane** [comment]
Thanks again! Closed by #3546
